yt/frontends/stream/tests/test_outputs_pytest.py

```diff
@@ -53,7 +52,9 @@ class TestStream:
     def test_load_empty_file(self):
         assert_raises(FileNotFoundError, load, "not_a_file")
         assert_raises(
-            YTUnidentifiedDataType, load, tempfile.mkstemp("empty_file", dir=os.getcwd())[1]
+            YTUnidentifiedDataType,
```

Comment:
Hey Jared, I removed this test a while ago, and reimplemented it in `yt/tests/test_load_errors.py`
I think this is another signal that we should not manage separate files (with the _pytest suffix) in this branch: this particular change is creating an unnecessary conflict with the master branch and I suspect there are more like this one happening within the PR.
I'm sorry, this is obviously a pain but I think that you should consider solving the root problem with how this branch develops before it gets out of control.
If you need help with defining a rebase/merge strategy to sanitize it, don't hesitate to reach out, I'm here to help :)

Reply:
@neutrinoceros I would love some help on coming up with a strategy! I've never actually done any rebasing before, so I'm open to learning!